### PR TITLE
Lock rpm

### PR DESCRIPTION
**Pull request by mray190**
maestro: v2.10.0
mbed-edge-examples: 0.10.0
k3s: v1.20.4+k3s1
golang-github-containernetworking-plugins: v0.8.4
mbed-edge-core: 0.13.0
mbed-edge-core-devmode: 0.13.0
devicedb: 1.10.0
maestro-shell: v2.7.1
global-node-modules: v1.0.0
edge-proxy: v1.0.0
mbed-fcc: 4.6.0
rallypointwatchdogs: v1.0.0
pe-utils: 2.0.6
kubelet: v1.0.0

### CODE REVIEW

**Review by CristianPrundeanuARM — APPROVED** [review]

File: maestro/rpm/maestro.spec [hunk 1]
```diff
@@ -112,5 +112,7 @@ install -vpm 0644 %{_filesdir}/maestro.service  %{buildroot}/%{_unitdir}
 %systemd_postun_with_restart maestro.service
 
 %changelog
+* Wed Jun 9 2021 Michael Ray <michael.ray@pelion.com> - 2.9.0-1
```
Comment (costanic) [inline maestro/rpm/maestro.spec:115]: The version here (2.9.0-1) is not consistent with the commit message (v2.10.0).

Comment (mray190) [reply maestro/rpm/maestro.spec:115]: Fixed. Thanks

**Comment by costanic** [comment]
rebased on dev.  I can now build amd64 locally

**Review by johnmichaelwelsh — APPROVED** [review]

**Comment by mray190** [comment]
CI passing all builds, just reporting an existing CI bug of:
`Archive JUnit-formatted test results`

Should be able to bypass CI at this point and merge

**Comment by costanic** [comment]
> CI passing all builds, just reporting an existing CI bug of:
> `Archive JUnit-formatted test results`
> 
> Should be able to bypass CI at this point and merge

Looks like this common error:

`stdout: {"account-id":"","edge-version":"unknown-release-branchless-uncommited-uninited","endpoint-name":"0177b3b1c16eca1ca4e742b703c00000","error_code":2,"error_description":"bad-request:Account device quota reached","internal-id":"","lwm2m-server-uri":"","status":"error"}`

**Pull request merged**